### ISSUE TITLE
entry-based URLs

### ISSUE THREAD

**Issue by kasunben** [reliefsupports/reliefsupports.org-v1#3]
So, people can share entries on social media.

**Comment by paarandika** [comment]
Is someone working on this? Otherwise, I can do this. (I think :))

**Comment by kasunben** [comment]
@paarandika Perfect. Go ahead. :)

**Comment by paarandika** [comment]
On it.

**Comment by kasunben** [comment]
@paarandika a small thing, can we go the URL patter something like this;
`http://reliefsupports.org/entry/{type}/{id}`

Ex-
```
http://reliefsupports.org/entry/needs/2
http://reliefsupports.org/entry/donations/5
```

**Comment by kasunben** [comment]
@paarandika Sorry!. Just saw you have done a good work on this. Perfect. Will check and merge.

**Comment by bhagyas** [comment]
Can the URL be even more shortened? Feels like `/entry/` is unnecessary.

Additionally, a random string would be better than a number, since a number can be misinterpreted.

**Comment by ssudaraka** [comment]
+1 for removing 'entry' from the URL.

However, keeping the ID is better than a random string because someone can quickly identify the post just by looking at the URL.

**Comment by bhagyas** [comment]
ID would be better, but we already have `count` and other metadata which are integers. Suggesting an alphanumeric shortened hash of the ID.

Also suggesting a change to the ID from a number to an alphanumeric random short string.

**Comment by kasunben** [comment]
@bhagyas @ssudaraka I think it's better we keep the`entry` part as an identifier. Just using as `rea__org/5` or like that kind of confusing and will narrow down the scope. Just a thought.

**Comment by kasunben** [comment]
Fixed with #136
